test/WebJobs.Script.Tests.Integration/Controllers/Keys/KeyManagementFixture.cs

```diff
@@ -33,9 +38,24 @@ protected override void RegisterDependencies(ContainerBuilder builder, WebHostSe
 
             SecretManagerMock = BuildSecretManager();
 
-            builder.RegisterInstance<ISecretManager>(SecretManagerMock.Object);
+            webHostBuilder.ConfigureServices(c => c.AddSingleton<ISecretManager>(SecretManagerMock.Object));
 
-            base.RegisterDependencies(builder, settings);
+            base.ConfigureWebHostBuilder(webHostBuilder);
+        }
+
+        public static ApiModel ReadApiModelContent(HttpResponseMessage response)
+        {
```

Comment:
Added this to workaround a change in behavior in Json.NET when deserializing types inheriting from JObject. Doesn't impact any product code paths.